app/src/main/java/edu/mills/heartoftaiwanese/data/TaiwaneseResult.kt

```diff
@@ -0,0 +1,8 @@
+package edu.mills.heartoftaiwanese.data
+
+import edu.mills.heartoftaiwanese.network.WebResultCode
+
+data class TaiwaneseResult(val resultCode: WebResultCode, val taiwanese: String? = null) {
```

Comment:
I actually have them separate instead of just Result for readability, so that it's easier to tell which result is which. The isChinese and isTaiwanese is just for sanity checking.